package.json

```diff
@@ -39,7 +39,7 @@
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-n": "^17.9.0",
     "prettier": "^3.2.5",
-    "typescript": "^5.5.4"
+    "typescript": "~5.5.4"
```

Comment:
Switched to only latest patch version bc 5.6 and above is not compatible with @typescript-eslint/estree.